Export libfdb c

atn34 opened the pull request:
Fixes #1483

atn34 commented:
It would be nice if this made it into 6.1. Please bear with me while I figure out how to make a pull request to 6.1

mpilman commented on build/cmake/package_tester/fdb_c_app/app.c:
```diff
@@ -0,0 +1,6 @@
+#include <foundationdb/fdb_c.h>
+
+int main(int argc, char* argv[]) {
+	return 0;
```
I think it would be better to also call at least one function from fdb_c - otherwise you won't catch any linker issues...

atn34 replied:
Done

AlvinMooreSr commented:
test this build

AlvinMooreSr commented:
test bindings

atn34 commented:
@AlvinMooreSr I changed the tests so that the docker images do not come with cmake or gcc preinstalled. I also changed the target branch back to master since I realized 6.1 isn't being built with cmake anyway. PTAL

atn34 commented:
We should probably also mark this as 6.2, since it fixes the install directory of the headers for the cmake generated client packages

atn34 commented:
@AlvinMooreSr Is there anything preventing this from getting merged?
